policyengine_us/variables/gov/states/mt/tax/income/deductions/cdcc/mt_child_dependent_care_expense_deduction.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class mt_child_dependent_care_expense_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana child dependent care expense deduction"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.MT
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.mt.tax.income.deductions.standard.mt_child_dependent_care_expense_deduction
+        agi = tax_unit("mt_agi", period)
+        care_expense = ("expenses_paid", period)
+        qualifying_child = ("qualifying_child", period)
```

Comment:
this variable does not exist, 
take the number of dependents who are either under 15 oder disabled